package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Dependent-Jaguar, MetroRail: Data Modeling & Database Design (LG Goal 126)",
   "main": "app.js",
   "scripts": {
-    "test": "mocha --watch dist/spec",
+    "test": "./node_modules/mocha/bin/_mocha",
```

Comment:
I think it's good for now. If you find yourself with a bunch of time at some point, we (or another coach) can try to debug it.